test/gridengine/setup_tool.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cp tool_conf.xml config
+cp galaxy.ini /etc/galaxy/galaxy.ini
```

Comment:
I setup 2 parameter for my version of galaxy.ini.
First one is `job_working_directory=...`
Second one is `cleanup_job=never`

At first one is I can setup via `GALAXY_CONFIG_JOB_WORKING_DIRECTORY`
And second one is for debug purpose (maybe).

So I can remove my own galaxy.ini.